packages/block-editor/src/components/inner-blocks/index.js

```diff
@@ -149,26 +149,53 @@ class InnerBlocks extends Component {
 			clientId,
 			hasOverlay,
 			__experimentalCaptureToolbars: captureToolbars,
+			forwardedRef,
 			...props
 		} = this.props;
 		const { templateInProcess } = this.state;
 
+		if ( templateInProcess ) {
+			return null;
+		}
+
+		const blockList = (
+			<BlockList
+				rootClientId={ clientId }
+				forwardedRef={ forwardedRef }
+				{ ...omit( props, [
+					'templateLock',
+					'isSmallScreen',
+					'block',
+					'blockListSettings',
+					'parentLock',
+					'isLastBlockChangePersistent',
+					'replaceInnerBlocks',
+					'__unstableMarkNextChangeAsNotPersistent',
+					'updateNestedSettings',
+					'__experimentalMoverDirection',
+					'allowedBlocks',
+					'template',
+				] ) }
```

Comment:
This component could benefit from a hooks rewrite, which will remove the need to omit props used by this component. I'll consider it out of scope for this PR though.

Reply:
So just to clarify here, the extra `...props` are needed for the "style" in this PR right?

Reply:
That's correct. Any remaining props on `InnerBlocks` should be passed down to the block wrapper.